cpp/src/arrow/flight/flight_test.cc

```diff
@@ -417,104 +441,71 @@ class MetadataTestServer : public FlightServerBase {
   }
 };
 
-template <typename T>
-class InsecureTestServer : public ::testing::Test {
```

Comment:
It seems InsecureTestServer disappeared. Is there a reason? Is the test folded somewhere else?

Reply:
It was only used in one place (original file line 446), so I replaced it with a direct subclass of `testing::Test` using the new helper function to create the server.